### PR TITLE
fix(@angular/*): add mappings for testing umd bundles

### PR DESCRIPTION
**Pull request by frederikschubert**
As [suggested](https://github.com/jspm/registry/issues/976#issuecomment-248043926) by @guybedford.
Closes https://github.com/jspm/registry/issues/976

### CODE REVIEW

File: package-overrides/npm/@angular/core@2.0.0-rc.1.json [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{
-  "meta": {
-    "src/*.js": {
-      "deps": ["reflect-metadata", "zone.js"]
-    }
-  }
-}
```
Comment (guybedford) [inline package-overrides/npm/@angular/core@2.0.0-rc.1.json]: This file can be left here, to support existing users. The new one without the prefix will still take preference.

File: package-overrides/npm/@angular/http@2.0.0-rc.1.json [hunk 2]
```diff
@@ -1,7 +0,0 @@
-{
-  "meta": {
-    "src/*.js": {
-      "deps": ["reflect-metadata", "zone.js"]
-    }
-  }
-}
```
Comment (guybedford) [inline package-overrides/npm/@angular/http@2.0.0-rc.1.json]: And also for this file.

File: package-overrides/npm/@angular/core@2.0.0-rc.1.json (hunk 1 above)
Comment (frederikschubert) [inline package-overrides/npm/@angular/core@2.0.0-rc.1.json]: Should I add the files back and then squash?

Comment (frederikschubert) [inline package-overrides/npm/@angular/core@2.0.0-rc.1.json]: Done.

**Comment by vigie** [comment]
This fix doesn't work for me as I have `defaultJSExtensions: true` so System ends up trying to load `./bundles/platform-browser-dynamic-testing.umd.js/index.js`

Is there any better way to correct this other than adding all my own local overrides?

**Pull request merged**